src/lib/db-listeners/db-problem-documents-listener.ts

```diff
@@ -75,47 +75,38 @@ export class DBProblemDocumentsListener extends BaseListener {
 
   private handleOfferingUser = (user: DBOfferingUser) => {
     if (!user?.self?.uid) return;
-    const { documents, user: currentUser, groups } = this.db.stores;
+    const { documents, user: currentUser } = this.db.stores;
     // uid should always be a string, but demo users with numeric uids have been encountered
     const isOwnDocument = String(user.self.uid) === currentUser.id;
-    // monitor problem documents
+
     if (isOwnDocument && (size(user.documents) === 0)) {
       documents.resolveRequiredDocumentPromiseWithNull(ProblemDocument);
     }
+
     forEach(user.documents, document => {
       if (!document?.documentKey || !document?.self?.uid) return;
       const existingDoc = documents.getDocument(document.documentKey);
       if (existingDoc) {
         this.db.updateDocumentFromProblemDocument(existingDoc, document);
       } else {
-        // both teachers and students listen to all problem documents
-        // but only teachers listen to all content.  students only listen
-        // to content of users in their group to reduce network traffic
-        const userInGroup = groups.userInGroup(document.self.uid, currentUser.latestGroupId);
-        // Local changes take precedence over remote changes
-        const monitor = isOwnDocument
-                          ? Monitor.Local
-                          : currentUser.isTeacher || userInGroup
-                            ? Monitor.Remote
-                            : Monitor.None;
-        this.db.createDocumentModelFromProblemMetadata(ProblemDocument, document.self.uid, document, monitor)
-          .then((doc) => {
+        this.db.createDocumentModelFromProblemMetadata(ProblemDocument, document.self.uid, document)
+          .then((docModel) => {
             if (isOwnDocument) {
-              documents.resolveRequiredDocumentPromise(doc);
-              syncStars(doc, this.db);
+              documents.resolveRequiredDocumentPromise(docModel);
+              syncStars(docModel, this.db);
             }
           });
       }
     });
-    // monitor planning documents
+
     if (isOwnDocument && (size(user.planning) === 0)) {
       documents.resolveRequiredDocumentPromiseWithNull(PlanningDocument);
     }
     forEach(user.planning, document => {
       if (!document?.documentKey || !document?.self?.uid) return;
       const existingDoc = documents.getDocument(document.documentKey);
       if (!existingDoc) {
-        this.db.createDocumentModelFromProblemMetadata(PlanningDocument, document.self.uid, document, Monitor.Local)
+        this.db.createDocumentModelFromProblemMetadata(PlanningDocument, document.self.uid, document)
```

Comment:
Note how this was Monitor.Local which meant that planning documents were not monitored.

Reply:
>Note how this was Monitor.Local which meant that planning documents were not monitored.

Right, because planning documents aren't shared between users the way problem documents are and we've never addressed the issue of monitoring network documents.